lib/plugins/stonith/external/ec2

```diff
@@ -53,15 +53,20 @@ quiet=0
 instance_not_found=0
 unknown_are_stopped=0
 
-action_default="reset"         # Default fence action
-ec2_tag_default="Name"	       # EC2 Tag containing the instance's uname
+action_default="reset"		# Default fence action
+ec2_tag_default="Name"		# EC2 Tag containing the instance's uname
+ec2_profile_default="Name"	# EC2 Profile containing the AWS's profile
```

Comment:
@ldevulder Could you change this to `default`? This is the default name usually

Reply:
Good catch, it's a copy/paste error, I'm changing this.